### PR TITLE
keymap_or_locale: handle tty3 properly

### PR DESCRIPTION
**Pull request by ggardet**
Try to fix aarch64, without breaking s390x

### CODE REVIEW

File: lib/susedistribution.pm [hunk 1]
```diff
@@ -570,6 +571,7 @@ sub console_nr {
     my $nr = 4;
     $nr = get_root_console_tty if ($name eq 'root');
     $nr = 5 if ($name eq 'log');
+    $nr = 3 if ($name eq 'extra');
```
Comment (okurz) [inline lib/susedistribution.pm]: Can we try to use backward mapping from the definition in init_consoles?

File: lib/susedistribution.pm [hunk 2]
```diff
@@ -392,6 +392,7 @@ sub init_consoles {
         $self->add_console('log-console',    'tty-console', {tty => 5});
         $self->add_console('displaymanager', 'tty-console', {tty => 7});
         $self->add_console('x11',            'tty-console', {tty => get_x11_console_tty});
+        $self->add_console('extra-console',  'tty-console', {tty => 3});
```
Comment (okurz) [inline lib/susedistribution.pm]: I would recommend to make the name explicit, e.g. just 'check_keyboard_layout'

Comment (foursixnine) [reply lib/susedistribution.pm]: I actually think makes more sense to have it named extra console instead :) check_keyboard_layout sounds too much like: "We need just this console, just for this.." which for the time being is true. But it's more an extra console in reality...

File: lib/susedistribution.pm (hunk 1 above)
Comment (ggardet) [reply lib/susedistribution.pm]: @okurz how would you do that?

File: lib/susedistribution.pm (hunk 2 above)
Comment (ggardet) [reply lib/susedistribution.pm]: @okurz Are you ok to stick to 'extra' for the name?

**Comment by ggardet** [comment]
I tested it successfully locally.

File: lib/susedistribution.pm (hunk 2 above)
Comment (okurz) [reply lib/susedistribution.pm]: well, alright

**Comment by okurz** [comment]
@mloviska as the test module maintainer for keymap_or_locale you might be interested. However, as this PR is now open for multiple months I will merge as is.

**Comment by mloviska** [comment]
LGTM

**Pull request merged**